checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```diff
@@ -0,0 +1,946 @@
+    name: Subject
+    path: checkpoint.quantum.firewall.subject
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source IP Port
+    path: checkpoint.quantum.firewall.src_ip_port
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Destination IP Port
+    path: checkpoint.quantum.firewall.dst_ip_port
+    source: log
+    type: string
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Total Bytes
+    path: checkpoint.quantum.firewall.bytes
+    source: log
+    type: integer
+    unit:
+      family: bytes
+      name: byte
+pipeline:
+  type: pipeline
+  name: Checkpoint Quantum Firewall
+  enabled: true
+  filter:
+    query: "source:checkpoint-quantum-firewall"
+  processors:
+    - type: service-remapper
+      name: Define `product` as the official service of the log
+      enabled: true
+      sources:
+        - product
+    - type: date-remapper
+      name: Define `time` as the official date of the log
+      enabled: true
+      sources:
+        - time
+    - name: Lookup for `severity` to `checkpoint.quantum.firewall.severity` field
+      enabled: true
+      source: severity
+      target: checkpoint.quantum.firewall.severity
+      lookupTable: |-
+        0, Info
+        1, Low
+        2, Medium
+        3, High
+        4, Critical
+        Info, Info
+        Low, Low
+        Medium, Medium
+        High, High
+        Critical, Critical
+      type: lookup-processor
+    - type: attribute-remapper
+      name: Map `subject` to `checkpoint.quantum.firewall.subject`
+      enabled: true
+      sources:
+        - subject
+      sourceType: attribute
+      target: checkpoint.quantum.firewall.subject
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `client_name` to `checkpoint.quantum.firewall.client_name`
+      enabled: true
+      sources:
+        - client_name
+      sourceType: attribute
+      target: checkpoint.quantum.firewall.client_name
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `action` to `checkpoint.quantum.firewall.action`
+      enabled: true
+      sources:
+        - action
+      sourceType: attribute
+      target: checkpoint.quantum.firewall.action
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `protocol` to `checkpoint.quantum.firewall.protocol`
+      enabled: true
+      sources:
+        - protocol
+      sourceType: attribute
+      target: checkpoint.quantum.firewall.protocol
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `bytes` to `checkpoint.quantum.firewall.bytes`
```

Comment:
Can you check if this fits into some of the standard attributes please ? https://docs.datadoghq.com/standard-attributes/?search=bytes

Reply:
According to checkpoint docs, bytes => Number of bytes received during a connection but there is no clarification about client to server or vice versa, hence we haven't mapped bytes to network.bytes_read or network.bytes_written. For reference: https://support.checkpoint.com/results/sk/sk144192

Reply:
Thanks !

In the documentation that you shared there seems to be `received_bytes` and `sent_bytes` attributes. Maybe those can be remapped to `network.bytes_read` and `network.bytes_written` ?

Reply:
We already have other attribute mapped to these standard attribute so this mapping doesn't seem feasible.

Reply:
Do those attributes appear in the same logs or in different types of logs ? 

In any case it is possible to map multiple attributes to a target, the order in the `sources` field will define which one has preference.

Reply:
Same logs. So the attributes that make the most sense are mapped, hence i don't think we need to map these